NuRadioMC/simulation/simulation.py

```diff
@@ -594,7 +595,11 @@ def run(self):
                             distance_cut_time += time.time() - t_tmp
 
                         self._raytracer.set_start_and_end_point(x1, x2)
+                        self._raytracer.set_shower_axis(self._shower_axis)
```

Comment:
I think it would be better to add ray tracing module specific settings. So something like
```
if self._cfg['propagation']['module'] == 'radiopropa':
    self._raytracer.set_shower_axis(self._shower_axis)
```
we might end up with more things that are specific to a certain module. We don't want to implement useless dummy function to all RT module for each special case. 

An alternative would be to add a generic interface like
```
ray_tracer.set_optional_parameter("shower_axis", shower_axis)
```
This can be defined in the base class, then check if the corresponding function is implemented in the RT module, and call it if it exists. In the analytic RT case if would do nothing, in the radiopropa case it would set the shower axis.

Reply:
I needed to set the shower axis in the radiorpropa tracer for a cut on the viewing angle to speed up the tracer. So I implemented it also in analytic tracer so the programme wouldn't not crash because I didn't think of a more elegant way